doc/content/verification_and_validation/ver-1jb.md

```diff
@@ -1,9 +1,142 @@
+- $C_M^0 = 1 \times 10^{25}$ atoms/m$^3$, which makes it equivalent to the initial trapped tritium concentration. This new case demonstrate TMAP8's ability to model the decay, trapping, detrapping, and diffusion of tritium better than when the concentration of mobile tritium is negligible, as in the other case.
+
+To limit the computational challenges related to the orders of magnitude difference in trapped and mobile tritium in the first case, we make the concentration dimensionless by dividing them by $1 \times 10^{25}$ atoms/m$^3$ and setting $\text{trap\_per\_free} = 1 \times 10^{25}$ (-). In the second case, when the concentration are equivalent, $\text{trap\_per\_free} = 1$ (-).
+
+## Analytical Solution
+
+The total inventory of T in atoms, $I_{tot} = I_M + I_T$ where $I_M$ and $I_T$ are the mobile and trapped tritium inventories, respectively, is given at any given time by
+
+\begin{equation}
+    I_{tot} = I_{tot}^0 \exp(-kt),
+\end{equation}
+
+where $I_{tot}^0 = I_M^0 + I_T^0$ atoms/m is the initial total inventory of tritium
+($I_M^0$ and $I_T^0$ are the initial mobile and trapped tritium inventories).
+Applying a mass balance over the system, the inventory of helium in atoms, $I_{He}$, is given by
+\begin{equation}
+    I_{He} = I_{tot}^0 \left[1- \exp(-kt) \right].
+\end{equation}
+
+### Results
+
+#### With a small concentration of mobile tritium compared to trapped tritium
+
+[ver-1jb_results_comparison_analytical_time_evolution_1] shows the TMAP8 predictions and how they compare to the analytical solution
+for the decay of tritium and associated growth of $^3$He in a distributed trap.
+TMAP8 matches the analytical solution, with a root mean square percentage error
+(RMSPE) of 1.29% and 0.35% for the $I_{tot}$ and $I_{He}$ concentration curves, respectively,
+and can also provide the trapped and mobile tritium concentrations.
+
+!media comparison_ver-1jb.py
+       image_name=ver-1jb_comparison_analytical_time_evolution.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1jb_results_comparison_analytical_time_evolution_1
+       caption=Comparison of TMAP8 predictions against the analytical solution for the decay of tritium and associated growth of $^3$He in a distributed trap with a small concentration of mobile tritium compared to trapped tritium.
+
+[ver-1jb_results_profile_1] shows the depth profile of the initial trapped atoms of tritium, the concentration of trapped atoms of
+tritium after 45 years, and the distribution of $^3$He at the end of that time across the distributed trap as predicted by TMAP8.
+
+Note that because of $^3$He is given a null diffusivity in this verification problem, the shape of the $^3$He does not broaden.
+This could be easily implemented with existing TMAP8 diffusive capabilities, as was shown in many other TMAP8 cases,
+including [ver-1d](ver-1d.md).
+
+!media comparison_ver-1jb.py
```

Comment:
Great catch in the legend!

Hmm, I'm not a fan of having the lower value in the y axis be lower than 0 since negative values makes no physical sense. I can add a sentence about the mobile concentration being close to 0 in this case.

Reply:
Yeah that works too. It's just good to mention something for clarity.

Reply:
Done.